interfaces/Portalicious/src/app/pages/project-payments/project-payments.page.ts

```diff
@@ -31,4 +37,15 @@ export class ProjectPaymentsPageComponent {
       detail: 'Functionality not implemented yet',
     });
   }
+  private paymentsQuery = injectQuery(
+    this.paymentApiService.getPayments(this.projectId),
+  );
+
+  payments = computed(() =>
```

Comment:
```suggestion
  paymentsSorted = computed(() =>
```

This would allow you to keep `payments = injectQuery` and not have a naming conflict